apps/speed.c

```diff
@@ -2688,8 +2688,8 @@ int speed_main(int argc, char **argv)
                    k, rsa_bits[k], rsa_results[k][0], rsa_results[k][1]);
         else
             printf("rsa %4u bits %8.6fs %8.6fs %8.1f %8.1f\n",
```

Comment:
Thanks for review.
I guess the extended precision doesn't matter any more. All other things being equal, for the purpose of the patch I usually go with minimum changes to ease up reviewer's work.